### DISCUSSION TITLE
Where is the webroot of the docker container (for let's encrypt)

### DISCUSSION THREAD

**Discussion by as4826** [wg-easy/wg-easy#571] [Q&A] [answered]
I'm trying to implement let's encrypt certificates for wg-easy web ui

 I ran this command `sudo certbot certonly --webroot` and now it's asking the webfoot for the domain

So my question is: Where is the webroot of the docker container?

**Comment by eandersons** [comment]
Why are you using the option `--webroot`?
On the host there obviously is no webroot for WireGuard Easy.

What webserver are you using?

**Reply by as4826** [reply]
Just following Certbot instructions https://certbot.eff.org/instructions?ws=other&os=ubuntufocal&tab=standard

So I should use option --standalone for wg-easy? 
If so, how to stop (and the run) web server? As I understood webserver is Express?


<img width="954" alt="image" src="https://github.com/wg-easy/wg-easy/assets/136320536/2ef2fac2-666d-4fd5-8ba5-367302d6741d">

**Reply by eandersons** [reply]
> If so, how to stop (and the run) web server? As I understood webserver is Express?

So that means that you are not using any reverse proxy?

I, for example, am using Nginx as a reverse proxy on my host, and getting certificates was as easy as `certbot --nginx` and then answering to some prompts.

**Comment by pheiduck** [comment, marked as answer]
We recommend to use Nginx Reverse Proxy:
eg: https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Caddy-SSL
